wrappers/swift/LogManager.swift

```diff
@@ -0,0 +1,160 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Wrapper over ODWLogManager which manages the telemetry logging system.
+public class LogManager {
+
+    /**
+    Initializes the telemetry logging system with the specified tenant token and custom configuration.
+
+    - Parameters:
+        - tenantToken: A `String` that contains the tenant token.
+        - config: A `Dictionary` that contains a custom configuration.
+
+    - Returns:
+        - A `Logger` instance.
+    */
+    public static func initForTenant(tenantToken: String, config: [AnyHashable: Any]? = nil) -> Logger? {
+        if let odwLogger = ODWLogManager.initForTenant(tenantToken, withConfig: config) {
+            return Logger(logger: odwLogger)
+        } else {
+            return nil
+        }
+    }
+
+    /**
+    Retrieves a new instance of `Logger` for logging telemetry events. Initializes the telemetry logging system,
+    if not already initialized, with the default configuration using the specified tenant token and source.
+
+    - Parameters:
+        - tenantToken: A `String` that contains the tenant token.
+        - withSource: A `String` that contains the name of the source of events.
+
+    - Returns:
+        - A `Logger` instance that points to the logger for the specified tenant token and source.
+    */
+    public static func loggerWithTenant(tenantToken: String, withSource source: String = "") -> Logger? {
+        if let odwLogger = ODWLogManager.logger(withTenant: tenantToken, source: source) {
+            return Logger(logger: odwLogger)
+        } else {
+            return nil
+        }
+    }
+
+    /**
+    Retrieves a new instance of `Logger` for logging telemetry events.
+    This method expects that the logging system has already been initialized.
+
+    - Parameters:
+        - tenantToken: A `String` that contains the tenant token.
+        - withSource: A `String` that contains the name of the source of events.
+        - config: A `Dictionary` that contains a custom configuration.
+
+    - Returns:
+        - A `Logger` instance that points to the logger for the specified tenant token and source.
+    */
+    public static func loggerWithTenant(tenantToken: String, withSource source: String, withConfig config: ODWLogConfiguration!) -> Logger?{
```

Comment:
Let's have this be something more Swifty, please; can we condense these into one function?

```
public static func logger(tenantToken: String?, source: String? = nil, config: ODWLogConfiguration? = nil) throws -> Logger {
    // instantiate and check for value 
    // if we don't have a valid logger, throw OneDSError.unableToCreateLogger
}
```

Reply:
Thought of doing it but since the  logic to instantiate the logger is bit different (along with error handling) on objc side for these two methods, I kept two separate methods in swift as well.
Condensing these would require a nil check and separate calls to these objc methods so i kept them separate to keep them cleaner.